src/cryptoadvance/specter/templates/components/explorer_select.jinja

```diff
@@ -0,0 +1,29 @@
+{% set chain_name = '' if (specter.chain == 'main' or specter.chain == 'regtest') else (
+    'signet' if specter.chain == 'signet' else 'testnet'
+)%}
+<select name="explorer" id="explorer" onchange="updateExplorer()">
+    <option value="https://mempool.space/" {% if specter.explorer == "https://mempool.space/" + chain_name %}selected{% endif%}>Mempool.space</option>
+    <option value="http://mempoolhqx4isw62xs7abwphsq7ldayuidyx2v2oethdhhj6mlo2r6ad.onion/{{ chain_name }}" {% if specter.explorer == "http://mempoolhqx4isw62xs7abwphsq7ldayuidyx2v2oethdhhj6mlo2r6ad.onion/" + chain_name %}selected{% endif%}>Mempool.space Tor hidden service</option>
+    <option value="https://blockstream.info/{{ chain_name }}" {% if specter.explorer == "https://blockstream.info/" + chain_name %}selected{% endif%}>Blockstream.info</option>
+    <option value="http://explorerzydxu5ecjrkwceayqybizmpjjznk5izmitf2modhcusuqlid.onion/{{ chain_name }}" {% if specter.explorer == "http://explorerzydxu5ecjrkwceayqybizmpjjznk5izmitf2modhcusuqlid.onion/" + chain_name %}selected{% endif%}>Blockstream.info Tor hidden service</option>
```

Comment:
Sorry, should have mentioned that before. Can't we return here the key only and not "expose" the url on the frontend? An improvement here would be to check on the backend whether the url is in one of the accpeted config-values but even better here would be to only return one of "MEMPOOL_SPACE", "MEMPOOL_SPACE_ONION", "BLOCKSTREAM_INFO", "BLOCKSTREAM_INFO_ONION" or "custom".
On the backend you can then check for "custom" and if it's not custom simply use the return-value as a key for config?

It would improve security and encapsulate business-logic to the controller rather then having that on the frontend.
What do you think?

Reply:
Just compared it with the "fee_estimation-selector". There, we're doing it more or less like that. Passing a key, resolving on the backend.

Reply:
I just realized that the block-explorer has some other flaws: Choosing e.b. "Memspace" works but it shows me as a custom-url. Ok, from a functional pointofview but confusing for the user.

So maybe we just don't fix this here but put it on the list of things to tidy up after the release.
This can then also be a bigger refactoring doing it consistently for both topics.